### PR TITLE
Add complete command to README and install docs

### PR DESCRIPTION
**Pull request by gsheni**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,17 +21,17 @@ EvalML is an AutoML library which builds, optimizes, and evaluates machine learn
 ```shell
 pip install evalml
 ```
-#### Install with Facebook's Prophet (time series)
+#### Add-ons
```
Comment (angela97lin) [inline README.md:24]: I'm super nit-picking but can we use ### instead? Seems like there's not a big difference between H4 and the bold below :)

File: docs/source/install.ipynb [hunk 2]
```diff
@@ -30,9 +29,21 @@
     "```\n",
     "\n",
     "#### Add-ons\n",
+    "You can install add-ons individually or all at once by running:\n",
+    "```bash\n",
+    "pip install evalml[complete]\n",
+    "```\n",
+    "\n",
+    "**Time Series Support** <br>\n",
```
Comment (angela97lin) [inline docs/source/install.ipynb:37]: I see <br> and \n here but it doesn't show up here?

image

**Review by angela97lin — APPROVED** [review]
Thanks for doing this, LGTM! Left just two nit-picky comments, feel free to ignore 😅

File: README.md (hunk 1 above)
Comment (gsheni) [reply README.md]: Fixed here
- https://github.com/alteryx/evalml/pull/2627/commits/13227609ed086d39a952912b9e5611525d1f6381

I also changed from bold to H4 headers for the subsections.

File: docs/source/install.ipynb (hunk 2 above)
Comment (gsheni) [reply docs/source/install.ipynb:37]: Fixed https://github.com/alteryx/evalml/pull/2627/commits/f208e624c4ee487d7b76dca6afef7f4be2496a79

**Pull request merged**